index.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     }
 
     this.versionChecker = new VersionChecker(this);
-    this.versionChecker.for('ember-cli', 'npm').assertAbove('0.2.0');
+    this.versionChecker.forEmber().assertAbove('0.2.0');
```

Comment:
This method will test for the Ember.js version, not Ember CLI as the original implementation is doing.